plugins/content_distribution/lib/kContentDistributionFlowManager.php

```diff
@@ -10,7 +10,7 @@ class kContentDistributionFlowManager extends kContentDistributionManager implem
 	 */
 	public function shouldConsumeChangedEvent(BaseObject $object, array $modifiedColumns)
 	{
-		if($object instanceof entry && $object->wasObjectSaved() && $object->getType() !== entryType::LIVE_STREAM)
+		if($object instanceof entry && $object->wasObjectSaved())
```

Comment:
@shpiNi  as part of our PHP coding standards, please don't create if-statements like this, use curly braces: 
if(//condition) 
{
}